1 table fpmembers.sql

```diff
@@ -0,0 +1,37 @@
+use FitPlexDB
+go 
+drop table if exists Member
+go 
+create table dbo.Member(
+     MemberID int not null primary key identity,
+     MemberFirstName varchar(25) not null
+          constraint c_Member_first_name_cannot_be_blank check(MemberFirstName <> ''),
+     MemberLastName varchar(25) not null 
+          constraint c_Member_last_name_cannot_be_blank check(MemberLastName <> ''),
+     MemberPhoneNum char(8) not null
+          constraint c_Member_phone_number_must_have_seven_digits check(MemberPhoneNum like '[1-9][0-9][0-9][-][0-9][0-9][0-9][0-9]'),
+          constraint u_Member_phone_number unique(MemberPhoneNum),
```

Comment:
Since we are going to reenter the member for each session they attend, if the phone num needs to be unique it will block the member from coming back to another session.

Reply:
Thank you for pointing that out. I removed the constraint.